versions.json

```diff
@@ -1,8 +1,8 @@
 {
   "design-system": [
     "7.0.0-beta.1",
+    "6.1.2",
     "6.0.1",
-    "6.0.0",
```

Comment:
This version seems redundant to list. Thought I'd remove it while making manual changes.